<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsDialog</class>
 <widget class="QDialog" name="StatsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>138</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="currentSessionSectionLabel">
     <property name="styleSheet">
      <string notr="true">font-weight:bold</string>
     </property>
     <property name="text">
      <string>Current Session</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="currentSessionSectionLayout" columnstretch="0,1">
     <property name="leftMargin">
      <number>18</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="currentUploadedLabel">
       <property name="text">
        <string>Uploaded:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="currentUploadedValueLabel">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="currentDownloadedLabel">
       <property name="text">
        <string>Downloaded:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="currentDownloadedValueLabel">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="currentRatioLabel">
       <property name="text">
        <string>Ratio:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="currentRatioValueLabel">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="currentDurationLabel">
       <property name="text">
        <string>Duration:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="currentDurationValueLabel">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="totalSectionSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="totalSectionLabel">
     <property name="styleSheet">
      <string notr="true">font-weight:bold</string>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="totalSectionLayout" columnstretch="0,1">
     <property name="leftMargin">
      <number>18</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="startCountLabel">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="totalUploadedLabel">
       <property name="text">
        <string>Uploaded:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="totalUploadedValueLabel">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="totalDownloadedLabel">
       <property name="text">
        <string>Downloaded:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="totalDownloadedValueLabel">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="totalRatioLabel">
       <property name="text">
        <string>Ratio:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="totalRatioValueLabel">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="totalDurationLabel">
       <property name="text">
        <string>Duration:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="totalDurationValueLabel">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>StatsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
